Weather App/UI/SettingsVC/SettingsCoordinator.swift

```diff
@@ -10,6 +10,8 @@ import UIKit
 
 class SettingsCoordinator: Coordinator {
     
+    var settingsData: SettingsData?
```

Comment:
Is a coordinator the best place to set/load some settings data? What is the main purpose of coordinators?

Reply:
Moved to SettingsViewModel